$NetBSD: patch-Makefile.am,v 1.1 2014/08/23 07:46:38 obache Exp $

* Prevent libfi to be linked with libintl.
* Avoid -no-undefined flags, or missing symbols might occur.

--- Makefile.am.orig	2014-03-26 12:46:44.000000000 +0000
+++ Makefile.am
@@ -67,19 +67,19 @@ flex_SOURCES = \
 	regex.c
 
 
-LDADD = lib/libcompat.la
+LDADD = lib/libcompat.la @LIBINTL@
 
 libfl_la_SOURCES = \
 	libmain.c \
 	libyywrap.c
 
-libfl_la_LDFLAGS = -no-undefined -version-info @SHARED_VERSION_INFO@
+libfl_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
 
 libfl_pic_la_SOURCES = \
 	libmain.c \
 	libyywrap.c
 
-libfl_pic_la_LDFLAGS = -no-undefined -version-info @SHARED_VERSION_INFO@
+libfl_pic_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
 
 noinst_HEADERS = \
 	flexdef.h \
@@ -124,7 +124,7 @@ SUBDIRS = \
 
 localedir = $(datadir)/locale
 AM_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -I$(top_srcdir)/intl
-LIBS = @LIBINTL@ @LIBS@
+LIBS = @LIBS@
 
 skel.c: flex.skl mkskel.sh flexint.h tables_shared.h
 	sed 's/m4_/m4postproc_/g; s/m4preproc_/m4_/g' $(srcdir)/flex.skl | $(m4) -P -DFLEX_MAJOR_VERSION=`echo $(VERSION)|cut -f 1 -d .` -DFLEX_MINOR_VERSION=`echo $(VERSION)|cut -f 2 -d .` -DFLEX_SUBMINOR_VERSION=`echo $(VERSION)|cut -f 3 -d .` | sed 's/m4postproc_/m4_/g' | $(SHELL) $(srcdir)/mkskel.sh  >skel.c
